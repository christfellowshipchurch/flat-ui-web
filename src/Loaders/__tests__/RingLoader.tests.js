import React from 'react'
import RingLoader from '../RingLoader'
import renderer from 'react-test-renderer'

describe('RingLoader', () => {
    it('renders the placeholder component', () => {
        const tree = renderer.create(<RingLoader />)

        expect(tree).toMatchSnapshot()
    })
})