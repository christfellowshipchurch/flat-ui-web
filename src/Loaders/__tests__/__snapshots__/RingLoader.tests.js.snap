// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RingLoader renders the placeholder component 1`] = `<div />`;
